src/js/pages/cart.js

```diff
@@ -52,7 +52,13 @@ export function loadProductCart(){
 }
 
 function renderTotalMoney(totalMoney){
-    document.querySelector('.total-money').innerHTML = formatMoney(totalMoney);
-    document.querySelector('.tax-money').innerHTML = formatMoney(totalMoney * 10 / 100);
-    document.querySelector('.total-payment').innerHTML = formatMoney(totalMoney * 110 / 100);
+    let money = document.querySelector('.total-money');
```

Comment:
Phân biệt giúp anh var let với const

Reply:
+ var: khai báo biến toàn cục
+ let: khai báo biến cục bộ
+ const: khai báo hằng số